examples/borer-circe-migration/build.sbt

```diff
@@ -0,0 +1,23 @@
+name := "borer-circe-migration"
```

Comment:
Hmmm so `examples/` consistute a completely separate sbt project? AFAICS it's not being built together with the main project, right?

Reply:
Is there any specific reason for that? I'd rather expect the examples to be a part of the main project, just never published

Reply:
> I'd rather expect the examples to be a part of the main project

From my experience, on the contrary. Example being [grpc-java](https://github.com/grpc/grpc-java) where [examples](https://github.com/grpc/grpc-java/tree/master/examples) are a separate project.

I don't see any merit in putting the examples in a main build tool, only drawbacks, including:
1. Examples are not self contained, they start depending on files outside of them
2. Figuring out simple dependencies requires analysing big buildfile
3. Clutters main build file with stuff we don't care about

Reply:
Ok, makes sense 👍🏻 
Either way, are the examples now being compiled within CI pipeline?

Reply:
That would require two things:
* Rewriting examples to scalatest (would like do anyway)
* Putting new command in .circleci
Implementing it is not a problem, but I would like to wait with final decision wether we run it with CI until we see what kind of impact it will have on CI execution time.

Reply:
Okay! pls give a try, I doubt it's gonna significantly increase the execution time, compared to what we've got now

Reply:
Rel, https://github.com/VirtusLab/akka-serialization-helper/issues/67

Reply:
1 minute - not great, not bad